lib/puppet-syntax.rb

```diff
@@ -7,12 +7,22 @@
 module PuppetSyntax
   @exclude_paths = []
   @future_parser = false
-  @hieradata_paths = ["**/data/**/*.*yaml", "hieradata/**/*.*yaml", "hiera*.*yaml"]
+  @hieradata_paths = [
```

Comment:
I moved these to new lines to make the diffs nicer, but I can do it in a separate PR if needed